qiskit/transpiler/_transpiler.py

```diff
@@ -54,32 +51,26 @@ def compile(circuits, backend,
         seed (int): random seed for simulators
         qobj_id (int): identifier for the generated qobj
         hpc (dict): HPC simulator parameters
-        skip_transpiler (bool): If True, bypass most of the compilation process and
-            creates a qobj with minimal check nor translation
-        skip_translation (bool): DEPRECATED. Use skip_transpiler instead.
 
     Returns:
         obj: the qobj to be run on the backends
 
     Raises:
-        QISKitError: if any of the circuit names cannot be found on the
-            Quantum Program.
+        TranspilerError: in case of bad compile options, e.g. the hpc options.
     """
-    if skip_translation:
```

Comment:
i agree about retiring `skip_translation`.

but `skip_transpiler` shouldn't be retired. it is a method of the high-level `compile()` function, and is convenient when a user wants to prevent any transformations on the circuit. this allows them to do that, without working directly with the PassManager

Reply:
But, considering the new API, is not `skip_transpiler=True` the same as passing an empty pass manager? If so, perhaps we can make `skip_transpiler` to use the _empty pass manager_ and, when everything is in place, log a warning saying it is better to pass the empty pass manager explicitly. What do you think?

Reply:
I think there may be a confusion here about the `compile()` methods. There are 2 compile functions: 
1. One is in `wrapper.compile()`. Here I am assuming the user doesn't want to know anything about the transpiler, just give a circuit and a backend, and the compilation happens in the default flow. Then if the user really doesn't want any transformation (this happens for example when you want faster compilation, and you are sure your circuit is compatible with the backend, e.g. a simulator), they can indicate this with the `skip_transpiler=True` flag. The majority of users use this high-level compile function.

2. The other is in `transpiler.compile()`. This compile actually takes a PassManager, and you can customize the passes you want to run on your circuit. This doesn't have `skip_translation`, because as you said, that's just equivalent to passing an empty pass manager.